src/components/modals/ChangeNetwork.tsx

```diff
@@ -30,15 +30,18 @@ export const ChangeNetworkModal: FC<IChangeNetworkModalProps> = ({
 	const NetworkName =
 		targetNetwork === config.MAINNET_NETWORK_NUMBER
 			? 'Ethereum'
-			: 'Gnosis Chain';
+			: 'Ethereum or Gnosis Chain';
```

Comment:
Remember that this modal will also be used on The Giving Block projects. If we show this message it won't make much sense. Can we make this modal receptive so for this case it shows Ethereum and for the others "Ethereum or Gnosis Chain"